### PR TITLE
dev - OpenBOR - joypad fix

### PR DESCRIPTION
**Pull request by Langerz82**
dev - OpenBOR - joypad fix

Current BUG:
Extra controllers need to enabled through game menus, and cannot use both DPAD and analog controls (one or the other).

Old issues:
In openBORff left analog is not working. - FIXED.
Not sure which keyboard assignment Attack 3 & 4 uses if any. - FIXED.

Related Issues:
https://github.com/EmuELEC/EmuELEC/issues/1080

### CODE REVIEW

**Comment by shantigilbert** [comment]
Is this good to merge? or what else needs to be done? (its marked as WIP)

File: packages/sx05re/emulators/openbor/package.mk [hunk 1]
```diff
@@ -42,6 +42,11 @@ makeinstall_target() {
     cp `find . -name "OpenBOR.elf" | xargs echo` $INSTALL/usr/bin/OpenBOR
     cp $PKG_DIR/scripts/*.sh $INSTALL/usr/bin
     chmod +x $INSTALL/usr/bin/*
+
     mkdir -p $INSTALL/usr/config/emuelec/configs/openbor
-	cp $PKG_DIR/config/master.cfg $INSTALL/usr/config/emuelec/configs/openbor/master.cfg
+		cp $PKG_DIR/config/master.cfg $INSTALL/usr/config/emuelec/configs/openbor/master.cfg
+		cp $PKG_DIR/config/masterff.cfg $INSTALL/usr/config/emuelec/configs/openbor/masterff.cfg
```
Comment (shantigilbert) [inline packages/sx05re/emulators/openbor/package.mk:48]: Why is this change needed? did you missed the package for openborff? 

https://github.com/EmuELEC/EmuELEC/blob/dev/packages/sx05re/emulators/openbor/openborff/package.mk

File: packages/sx05re/emulators/openbor/scripts/openbor.sh [hunk 2]
```diff
@@ -23,16 +23,17 @@ SAVES="${CONFIGDIR}/Saves"
...
+if [ ${OB} = "OpenBORff" ]; then
+    ln -sf "${CONFIGDIR}/masterff.cfg" "${SAVES}/${pakname}.cfg"
+else
+    ln -sf "${CONFIGDIR}/master.cfg" "${SAVES}/${pakname}.cfg"
+fi
+#	fi
 
 # We start the fake keyboard
-gptokeyb openbor &
+gptokeyb -c /emuelec/configs/gptokeyb/${OB}.gptk openbor &
```
Comment (shantigilbert) [inline packages/sx05re/emulators/openbor/scripts/openbor.sh:36]: Why are 2 identical gptk files used?

**Comment by Langerz82** [comment]
@shantigilbert 
Ooops I missed that openborff seperate package and updated.
Also I see your point only one gptokeyb profile is needed as they are identical.
I updated the PR to reflect the changes.

**Comment by Langerz82** [comment]
I double check that the gptokeyb script copies into storage in an update, and it does so I'm ok with this to be pushed.

**Review by shantigilbert — APPROVED** [review]

**Comment by shantigilbert** [comment]
Thank you
